### PR TITLE
fix istio injection field visibility

### PR DESCRIPTION
**Pull request by rizarsyi**

### CODE REVIEW

File: pygluu/kubernetes/gui/views/wizard.py [hunk 1]
```diff
@@ -824,8 +842,6 @@ def couchbase():
         data["COUCHBASE_NAMESPACE"] = form.couchbase_namespace.data
         data["COUCHBASE_CLUSTER_NAME"] = form.couchbase_cluster_name.data
         data["COUCHBASE_URL"] = form.couchbase_url.data
-        data["COUCHBASE_SUPERUSER"] = form.couchbase_superuser.data
```
Comment (moabu) [inline pygluu/kubernetes/gui/views/wizard.py:827]: why is this being removed ?

File: pygluu/kubernetes/gui/views/wizard.py [hunk 2]
```diff
@@ -824,8 +842,6 @@ def couchbase():
...
-        data["COUCHBASE_SUPERUSER_PASSWORD"] = form.couchbase_superuser_password.data
```
Comment (moabu) [inline pygluu/kubernetes/gui/views/wizard.py:828]: ?

File: pygluu/kubernetes/gui/views/wizard.py [hunk 3]
```diff
@@ -864,12 +880,6 @@ def couchbase():
         else:
             form.couchbase_password_confirmation.data = settings.get("COUCHBASE_PASSWORD")
 
-        if not settings.get("COUCHBASE_SUPERUSER_PASSWORD"):
```
Comment (moabu) [inline pygluu/kubernetes/gui/views/wizard.py:867]: ?

**Review by moabu — CHANGES_REQUESTED** [review]

File: pygluu/kubernetes/gui/views/wizard.py (hunk 1 above)
Comment (rizarsyi) [reply pygluu/kubernetes/gui/views/wizard.py:827]: Ah sorry, i think the ide not refreshed when i checkout new branch..

**Comment by rizarsyi** [comment]
I'm closing this PR

**Pull request closed without merging**